### PR TITLE
Buf tour updates

### PR DESCRIPTION
**Pull request by lucperkins**
This PR makes some readability updates to the Buf tour. I only covered a few pages here, as I'd prefer to get some initial pushback from folks rather than pressing on. Most notable changes:

* The current tour is a bit confused between "you" and "we." These changes unify the docs around "you."
* I've adopted Docusaurus' built-in syntax for specifying CLI commands and outputs. This enables users to use the **Copy** button to copy only the command and not the output.

### CODE REVIEW

File: docs/tour/configure-and-build.md [hunk 1]
```diff
@@ -103,9 +103,13 @@ in this project:
 
 ```terminal
 $ buf build --exclude-source-info -o -#format=json | jq '.file[] | .package'
+---
```
Comment (mfridman) [inline docs/tour/configure-and-build.md:106]: Nice, you discovered this!

We added this functionality _after_ the majority of the tour has been written.

Although subjective, I quite like how the example output appears below the command. We made an effort to make sure with `terminal` the commands are copy/pastable.

Comment (lucperkins) [reply docs/tour/configure-and-build.md:106]: @mfridman That seems pretty much objectively better to me 😄

File: docs/tour/configure-and-build.md [hunk 2]
```diff
@@ -6,12 +6,12 @@ title: 1 Configure and Build
...
-You're expected to be in the `start` directory of the [https://github.com/bufbuild/buf-tour.git](https://github.com/bufbuild/buf-tour.git)
-repository. From here, move into the `petapis` directory, which contains the pet store's
-`.proto` files.
+Clone the [`bufbuild/buf-tour`][tour_repo] repository from GitHub and navigate to the
+`petapis` directory, which contains the pet store's `.proto` files:
 
 ```terminal
-$ cd petapis
+$ git clone https://github.com/bufbuild/buf-tour.git
+$ cd buf-tour/start/petapias
```
Comment (amckinney) [inline docs/tour/configure-and-build.md:14]:
```suggestion
$ cd buf-tour/start/petapis
```

File: docs/tour/configure-and-build.md [hunk 3]
```diff
@@ -93,18 +93,31 @@ Before we continue, let's verify that everything is set up properly:
...
+
+Package name | Meaning
+:------------|:-------
+`google.protobuf` | A dependency on the [Well-Known Types](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf)
+`google.type` | A dependency on a Google API
+`pet.v1` | The pet store API itself
+
+We'll come back to the `pet.v1` package later.
+
+
```
Comment (amckinney) [inline docs/tour/configure-and-build.md:122]: nit: There might be an extra space here.

File: docs/tour/lint-your-api.md [hunk 4]
```diff
@@ -35,15 +27,26 @@ breaking:
...
+---
+{"path":"google/type/datetime.proto","start_line":17,"start_column":1,"end_line":17,"end_column":21,"type":"PACKAGE_VERSION_SUFFIX","message":"Package name \"google.type\" should be suffixed with a correctly formed version, such as \"google.type.v1\"."}
+{"path":"pet/v1/pet.proto","start_line":44,"start_column":10,"end_line":44,"end_column":15,"type":"FIELD_LOWER_SNAKE_CASE","message":"Field name \"petID\" should be lower_snake_case, such as \"pet_id\"."}
+{"path":"pet/v1/pet.proto","start_line":49,"start_column":9,"end_line":49,"end_column":17,"type":"SERVICE_SUFFIX","message":"Service name \"PetStore\" should be suffixed with \"Service\"."}
+```
+
 ## 3.1 Lint Exceptions {#lint-exceptions}
 
-The `DEFAULT` lint category failures we're seeing come from following rules:
+The [`DEFAULT`](/lint/rules#default) lint category failures we're seeing come from these rules:
```
Comment (amckinney) [inline docs/tour/lint-your-api.md:42]: With the change from `we` to `you`, should we update `we're` to `you're` or `you are`? That, or just drop it entirely.
```suggestion
The [`DEFAULT`](/lint/rules#default) lint category failures come from these rules:
```

**Review by amckinney — APPROVED** [review]
These is great 👏

File: docs/tour/lint-your-api.md (hunk 4 above)
Comment (lucperkins) [reply docs/tour/lint-your-api.md:42]: Good catch

**Pull request merged**